docs/examples/junit4/generic/src/test/java/generic/HostPortExposedTest.java

```diff
@@ -50,14 +50,15 @@ public static void tearDown() throws Exception {
     }
 
     @Rule
-    public BrowserWebDriverContainer browser = new BrowserWebDriverContainer().withCapabilities(new ChromeOptions());
+    public BrowserWebDriverContainer<?> browser = new BrowserWebDriverContainer<>()
+        .withCapabilities(new ChromeOptions());
 
     @Test
     public void testContainerRunningAgainstExposedHostPort() {
         // useHostExposedPort {
         final String rootUrl = String.format("http://host.testcontainers.internal:%d/", localServerPort);
 
-        final RemoteWebDriver webDriver = browser.getWebDriver();
+        final RemoteWebDriver webDriver = new RemoteWebDriver(this.browser.getSeleniumAddress(), new ChromeOptions());
```

Comment:
We should also change our docs/code examples to incorperate this pattern (can be follow-up PR of course). Current docs still access the web driver directly: 
https://www.testcontainers.org/modules/webdriver_containers/